.github/label-actions.yml

```diff
@@ -64,6 +64,13 @@
 
     If you're running self-hosted, run with `LOG_LEVEL=debug` in your environment variables and search for whatever dependency/branch/PR that is causing the problem.
 
+    Add steps how to sufficiently enable logging.
```

Comment:
Hmm, maybe we should drop the text instructions here, and link to [Renovate docs, troubleshooting, self-hosted section](https://docs.renovatebot.com/troubleshooting/#self-hosted) instead?

That way we have _one place_ for the information, and people will hopefully read the docs and complain if it's not clear. :smile:

Reply:
Good idea – however, I still would not know how to exactly create the (requested) plain log file from it.

As I already mentioned in the thread given as context, I was adopting the settings from the [official Renovate runner for GitLab](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/renovate.gitlab-ci.yml?ref_type=heads#L8) – and the resulting log file in ndjson format was frowned upon.

Reply:
I dropped the explanation in the bot comment, and instead link to the Renovate docs. I've moved the draft text to the troublehooting docs.